_data/amy.yml

```diff
@@ -1292,9 +1292,9 @@ airports:
 - country: br
   instructors:
   - {name: Filipe Fernandes, user: fernandes_filipe}
-  latitude: -12.910833
-  longitude: -38.330833
-  name: Salvador de Bahia
+  latitude: -27.59
```

Comment:
This info should be updated from AMY.

Reply:
Thanks! I'll update the PR to reflect that and check what I can do with AMY.

Reply:
In AMY it will be set by your airport code.

Reply:
Thanks! I updated AMY a few days ago. (Sorry for my ignorance but I AMY did not existed when I took the instructor training :grimacing:)

A related question regarding AMY. I noticed that there are some fields pending on my profile, that is odd b/c I do recall doing most those back in 2014.

![photo4933784363942914118](https://user-images.githubusercontent.com/950575/40783197-1e048e98-64b8-11e8-8874-ddc053e24434.jpg)

Reply:
This is a bug experienced by older instructors who we didn't track the steps for. Can you please send a note to team@carpentries.org and the team will fix your profile for you.

Reply:
Will do. thanks!